src/utils/pdf.js

```diff
@@ -0,0 +1,18 @@
+const puppeteer = require('puppeteer');
+
+module.exports = async function(html, options) {
```

Comment:
Make this a named export. We might need more utilities related to pdf in the future.

Reply:
I made this a named export, please review it

Reply:
Nice